### ISSUE TITLE
Problem to see the graphs for one Chrome navigator

### ISSUE THREAD

**Issue by heloisethero** [medialab/minivan#46]
For one computer (Mac OS Sierra 10.12.6) and a Chrome browser (Version 83.0.4103.97 (Official Build) (64-bit)), the iframes made with Minivan are all grey, and the graphs cannot be seen: 
![Screen Shot 2020-06-16 at 12 08 13](https://user-images.githubusercontent.com/26120765/84762537-345db600-afcb-11ea-8908-8cae490fdf2b.png)

This web page is normally like that: 
![Screenshot from 2020-06-16 12-13-19](https://user-images.githubusercontent.com/26120765/84762690-6d962600-afcb-11ea-91bd-98e59ca6caeb.png)

**Comment by Yomguithereal** [comment]
Hello @heloisethero, can you add the link to the article for easy debugging :) ?

**Comment by heloisethero** [comment]
Yes of course: https://medialab.sciencespo.fr/actu/les-infox-sur-le-covid-sous-surveillance/

**Comment by Yomguithereal** [comment]
The iframe's url for reference: https://medialab.github.io/minivan/#/embeded-network/?bundle=https:%2F%2Fraw.githubusercontent.com%2Fmedialab%2Fwebclim_analyses%2Fmaster%2Fgraph%2FCovid-19.json&color=type&lockNavigation=false&name=Covid-19%E2%88%B6%3D1.119744&showLink=true&size=nb_fake_news_shared&x=0.49811716909854986&y=0.500048141062655

**Comment by Yomguithereal** [comment]
I cannot reproduce the issue on a Chrome v83. What version is your Chrome?

**Comment by heloisethero** [comment]
Chrome Version 83.0.4103.97

**Comment by Yomguithereal** [comment]
The plot thickens. Which OS?

**Comment by heloisethero** [comment]
Mac OS Sierra 10.12.6
